### PR TITLE
fixed status page responsive

### LINKED ISSUE #397: fix the stats page
#329  breaked stat page for mobiles 
fix it and also try to make the stats side by side in the desktop rather than making it stretched

### PR DESCRIPTION
**Pull request by Rehan52004**
## Description

Make Status Page Responsive
Fixes #397 

## Screenshots / GIFs (if applicable)

paws-7
paws-8
paws-9

Desktop Screenshots: 

paws-10 - Copy
paws-11

## Checklist:
- [Yes] I have performed a self-review of my code
- [ ] I have added/updated relevant documentation (if needed)
- [Yes] I have tested the changes locally and they function as expected
- [Yes] I have ensured my code follows the project's coding standards

### CODE REVIEW

File: src/pages/ngo/StatPage/LineChart.jsx
Comment (codewarnab) [inline src/pages/ngo/StatPage/LineChart.jsx:1]: revert back all the double quotes to single quotes you changed ?

Comment (codewarnab) [inline src/pages/ngo/StatPage/LineChart.jsx:1]: also i don't  get it  why their so much extra formatting , please try to change the format as previously . 
please keep in mind when you are contributing to opensource your unnecessary changes should as  less possible

File: src/pages/ngo/StatPage/StatsMain.jsx
Comment (codewarnab) [inline src/pages/ngo/StatPage/StatsMain.jsx:1]: revert the format as previously

**Review by codewarnab — CHANGES_REQUESTED** [review]

**Comment by codewarnab** [comment]
also i  overlooked some things in screenshot previously 
1. remove extra space below  the heading ( statistics)
2. heat map of reported animal should and average response time  this headings should not be wrappend

**Comment by Rehan52004** [comment]
Ok
